examples/weird-types.flow.yaml

```diff
@@ -0,0 +1,92 @@
+collections:
+  - name: weird-types/optionals
+    schema:
+      type: object
+      properties:
+        theKey: { type: string }
+        string:
```

Comment:
nit: mix of ``property:\ntype: string`` here vs ``property: { type: string }`` below.

I've been using inline {} whenever it's a single-property object, if that sounds good to you for standardization.